sys/decl.go

```diff
@@ -391,6 +391,8 @@ type IntType struct {
 	TypeCommon
 	TypeSize uintptr
 	Kind     IntKind
+	IsRange  bool
+	IntRange []uintptr
```

Comment:
Using a slice here is confusing. It does not say how many entries must be in the slice, while we always expect a fixed number of elements. It can also have wrong number of elements, so there is also a temptation to check that a slice has correct number of arguments, but it will only unnecessary bloat code.
Also range can have negative begin/end.
Just use:
```
int64 rangeBegin
int64 rangeEnd
```